src/jaegertracing/Tracer.cpp

```diff
@@ -85,6 +86,7 @@ Tracer::StartSpanWithOptions(string_view operationName,
                 flags |=
                     (static_cast<unsigned char>(SpanContext::Flag::kSampled) |
                      static_cast<unsigned char>(SpanContext::Flag::kDebug));
+                samplerTags.push_back(Tag(kJaegerDebugHeader, parent->debugID()));
```

Comment:
is there a unit test which can check that?

Reply:
Hey @yurishkuro thanks for the review; tests added in 7d81527